dashboard/app/views/reports/lookup_section.html.haml

```diff
@@ -0,0 +1,17 @@
+%h1 Lookup section
+
+- if flash[:error]
+  #error_explanation
+    %h2= flash[:error]
+
+= form_tag(lookup_section_url, method: :post) do
+  Enter the section code:
+  %br/
+  = text_field_tag 'section_code'
+  %br/
+  = submit_tag 'Go!'
+
+- if @section
+  %p= "Section code: #{@section.code}"
+  %p= "Name: #{@section.name}"
+  %p= "Owner: #{@section.user.email}"
```

Comment:
we should probably just add a link to the section page in the teacher dashboard at this point.
The format of the url is: http://code.org/teacher-dashboard#/sections/12345
(an admin can view any section in teacher dashboard)